pages/api/team.js

```diff
@@ -0,0 +1,90 @@
+import AirtablePlus from 'airtable-plus'
```

Comment:
Please don't use airtable plus. It has not been updated for 5 years or so. Use the airtable package

Reply:
https://www.npmjs.com/package/airtable

Reply:
Other pages on the site use airtable plus - I'll change it here, but another PR could be made for the other routes.

Reply:
ah, I did not know that but I suppose we should switch to airtable since that package is deprecated

Reply:
Can you open a issue?